backend/shared/setup.py

```diff
@@ -21,9 +21,7 @@
     url="https://github.com/City-of-Helsinki/yjdh",
     author="City of Helsinki",
     author_email="dev@hel.fi",
-    install_requires=[
-        "Django",
-    ],
+    install_requires=["Django", "mozilla_django_oidc"],
```

Comment:
Do we need to add `django_auth_adfs` and `django-extensions` here as well?

Reply:
What is the use of `django-extensions` there? I guess it's to enable the local https:// server? Is there any other purpose? If so I think we can just keep it in project requirement-dev.
I fixed `django_auth_adfs` btw

Reply:
It would be for the `HourlyJob`: https://github.com/City-of-Helsinki/yjdh/pull/115/files#diff-f2014c778aaa7c004551dfb12563b6d347deffa2344c1ba22de99f9e4abddaadR2